eth/transaction.go

```diff
@@ -127,3 +127,16 @@ func (t *Transaction) MarshalJSON() ([]byte, error) {
 	u := unknown(*t)
 	return json.Marshal(&u)
 }
+
+// require values for valid tx
+func NewTransaction(nonce int64, gasPrice int64, gasLimit int64, toAddr string, value int64, data []byte) (*Transaction, error) {
```

Comment:
There's a couple issue with this function:

- many of those quantity values can exceed the storage of a int64.
- data can't be cast to string, it needs to be converted to hex
- Public functions should have a doc comment of the form `// <FunctionName> does X Y Z` rather than an internal comment.

What's the goal of this function?

Reply:
The goal of the function is to make it easier to create/sign a transaction. These are the minimum inputs needed to send a transaction so I aded a helper function to create a transaction for signing. (I will fix the int and string stuff)